.appveyor.yml

```diff
@@ -40,12 +40,16 @@ before_build:
     --sub-ref=%APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%:has=/sync$
     --print
   - python --version
-  - pip install -r tools/cbse/requirements.txt --upgrade
+  - python -m pip install -r tools/cbse/requirements.txt --upgrade
```

Comment:
Let's leave comments for why things like pip don't work.

Reply:
It does work fine here (at the moment) but this is just more reliable. I added details in the commit message.